### DISCUSSION TITLE
Rename Enterprise to Organization

### DISCUSSION THREAD

**Discussion by lecoqlibre** [datafoodconsortium/standarddocumentation#8] [Ideas]
Wouldn’t it be better to talk about Organization rather than Enterprise?

For example, an association is not really an enterprise.

**Comment by RaggedStaff** [comment]
I agree completely.

In fact, I'm wondering if there are any compelling reasons we're not referencing foaf for Agent data?

**Reply by lecoqlibre** [reply]
I you are asking why we don't use a `foaf:Agent` instead of a `dfc-b:Agent`, a `foaf:Person` instead of a `dfc-b:Person` and so on, AFAIK it's because a choice has been made since the beginning of the project. Indeed, the first ontologists have wanted the DFC to have its own vocabulary which was supposed to be aligned to other vocabularies later. I guess this was to have more facility and flexibility to build the standard seeing what was needed at the first place.

We asked Florence and Baptiste about it and they said that there is not good or bad choice. Although I guess that doing so makes more work to align vocabularies.

So we did continue to define our own vocabulary to stay consistent with what has been previously done.

**Comment by RaggedStaff** [comment]
@lecoqlibre @Alcoz  Wondering if we want to include this in v2 changes ? :thinking: 

Or do we want to minimise actual ontology changes, when we're changing every single URI ?

**Reply by lecoqlibre** [reply]
I vote yes to include this in the V2. 

I don't know why I created this discussion in the standard repo. This should be in the ontology repo.

Should we transfer this discussion to the ontology repo, create an issue from the discussion and then assign it to the next major ontology Milestone?
